lib/Conversion/FIRRTLToRTL/LowerToRTL.cpp

```diff
@@ -1427,19 +1427,17 @@ LogicalResult FIRRTLLowering::visitDecl(MemOp op) {
   // While it is possible for there to be more than one subfield per port, they
```

Comment:
This comment should be removed now that you've defined away the representational problem. :)

Reply:
Will be updated in rebase.